daml/Tests/Distribution/Syndication/Origination.daml

```diff
@@ -65,9 +65,9 @@ origination Parties{..} = do
 
   submitMulti [operator, bondRegistrar] [] do createCmd HolidayCalendar with operator; provider = bondRegistrar; calendar = fedCalendar; observers = fromList [issuer]
 
-  (bond1Desc, _) <- originateBond operator bondRegistrar issuer public bond1
-  (bond2Desc, _) <- originateBond operator bondRegistrar issuer public bond2
-  (bond3Desc, _) <- originateBond operator bondRegistrar issuer public bond3
-  (swap1Desc, _) <- originateSwap operator bondRegistrar issuer public swap1
+  (bond1Desc, _) <- originateBond operator bondRegistrar issuer [public, payingAgent] bond1
```

Comment:
might be problematic, because we need to know the paying agent at the time of origination (where the deal/tranches might not yet be created). I'll think if there's another solution to this. Good to be merged like this for now.

Reply:
Yes agree! We actually have another Jira in the pipeline https://digitalasset.atlassian.net/browse/FL-173 might need to rethink about what you mention.